src/GeographicalFeatures/AdministrativeAreas/index.js

```diff
@@ -155,7 +305,13 @@ class AdministrativeAreas extends Component {
       searchQuery,
       total,
     } = this.props;
-    const { showFilters, isEditForm, cached } = this.state;
+    const {
+      showFilters,
+      isEditForm,
+      cached,
+      notificationBody,
+      showNotificationForm,
+    } = this.state;
     return (
       <div className="AdministrativeAreas">
```

Comment:
This div also should be removed along with styles in styles.css can you check modules which are refactored already